data/index.md

```diff
@@ -0,0 +1,39 @@
+# Data Pipeline
+> tl;dr distributed systems are hard.
+
+All **write** attempts in the everydayhero ecosystem are served by a leader system. **Reads** aren't quite so straightforward. Let's begin by considering the categories of reads as we see them, and the expectations for each.
+
+- Public API (JSON)
+    - Presents data that spans more than one leader system
+    - Performs many aggregations (statistics)
+- Traditional reports (PDF / CSV / XLSX)
+    - Generally a time-window is specified
+    - Explicit latency is tolerable, within reason
+    - Given that these are frequently re-purposed by consumers, consistency (especially for $$$) is expected
+- Neartime / streaming / data-subscriptions
+    - Some of our more sophisticated customers (who are frequently high-value) expect near-time feedback of the performance of their campaigns.
```

Comment:
True, but the converse is not always so: some of our high value clients are not at all sophisticated (e.g. 40HF)